public/docs/ts/latest/guide/dependency-injection.jade

```diff
@@ -95,10 +93,10 @@ include ../../../../_includes/_util-fns
   If someone extends the `Engine` class, that is not `Car`'s problem.
 .l-sub-section
   :marked
-    The consumer of `Car` has the problem. The consumer must update the car creation code to
-    something like:
+    The _consumer_ of `Car` has the problem. The consumer must update the car creation code to
+    something like this:
 
-  +makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation-with-param', 'Super Car')(format=".")
+  +makeExample('dependency-injection/ts/app/car/car-creations.ts', 'car-ctor-instantiation-with-param', '')(format=".")
```

Comment:
But it's _not_ a derived or different class. It's just two different ways of calling the same constructor.

Either way, I don't see how the title helps you grasp the difference. Maybe if I highlighted the constructor call, that would be more helpful.